packages/runtime/src/useCases/transport/tokens/TokenMapper.ts

```diff
@@ -32,6 +32,22 @@ export class TokenMapper {
         };
     }
 
+    public static toEmptyTokenDTO(token: EmptyToken, ephemeral: boolean): EmptyTokenDTO {
+        const backboneBaseUrl = Container.get<ConfigHolder>(ConfigHolder).getConfig().transportLibrary.baseUrl;
+        const reference = token.toTokenReference(backboneBaseUrl);
+
+        return {
+            id: token.id.toString(),
+            expiresAt: token.expiresAt.toString(),
+            passwordProtection: PasswordProtectionMapper.toPasswordProtectionDTO(token.passwordProtection),
+            reference: {
+                truncated: reference.truncate(),
+                url: reference.toUrl()
+            },
+            isEphemeral: ephemeral
+        };
+    }
```

Comment:
Shouldn't there be an EmptyTokenMapper for this method?

Reply:
Isn't an empty token also a token?